src/main.rs

```diff
@@ -83,8 +83,8 @@ async fn handle_request(
         ..
     } = parts;
 
-    let trimmed_path = if uri.path().starts_with("/api/v6") {
-        uri.path().replace("/api/v6", "")
+    let trimmed_path = if uri.path().starts_with("/api/v8") {
```

Comment:
Can this get moved to a constant?

Reply:
You can use `concat!` to make sure that the string is made at compile-time and not at run-time

Reply:
it's now using the version exposed by twilight_http, so it can not be a const or made by concat